browser/mapping_test.go

```diff
@@ -253,3 +253,9 @@ type keyboardAPI interface { //nolint: unused
 	Type(text string, opts goja.Value)
 	Up(key string)
 }
+
+// touchscreenAPI is the interface of a touchscreen.
+// TODO: map this to page.GetTouchscreen().
```

Comment:
See: 5528008

Reply:
I see you updated the [comment](https://github.com/grafana/xk6-browser/blob/f0c037e4fd528ab79e6be2b53134212ea3b3067c/browser/mapping_test.go#L537-L540), but I'm not sure I'm understanding this properly.

With these changes for `page.[GetKeyboard()|GetMouse()|GetTouchscreen()`, because now we return the concrete type instead of the interfaces, aren't we exposing properties and methods to the JS layer that we do not want to expose?

Reply:
Yes, we expose their properties, but we haven't been testing what methods they return. For that, we need to check what methods they return using their API interfaces defined in the mapping layer. For example:

https://github.com/grafana/xk6-browser/blob/f0c037e4fd528ab79e6be2b53134212ea3b3067c/browser/mapping.go#L495

`keyboard` returns `*Keyboard`, but `page` mapping test doesn't test `*Keyboard` methods:

https://github.com/grafana/xk6-browser/blob/f0c037e4fd528ab79e6be2b53134212ea3b3067c/browser/mapping_test.go#L138-L147

The test should use the `keyboardAPI` interface. Even if you change the methods in `keyboardAPI`, the `page` mapping test would still pass.

Reply:
I created an issue for this https://github.com/grafana/xk6-browser/issues/1061.